src/aics-image-viewer/components/TfEditor/index.tsx

```diff
@@ -72,14 +79,36 @@ const TF_GENERATORS: Record<string, (histogram: Histogram) => Lut> = {
   resetXF: (_histo) => new Lut().createFullRange(),
 };
 
+// *---*
+// |   |
+// |   |
+//  \ /
+//   *
+// width: 0.65 * height; height of rectangle: 0.6 * height; height of triangle: 0.4 * height
+const sliderHandleSymbol: d3.SymbolType = {
+  draw: (context, size) => {
+    // size is symbol area in px^2
+    const height = Math.sqrt(size * 1.9);
+    const triangleHeight = height * 0.4;
+    const halfWidth = height * 0.325;
+
+    context.moveTo(-halfWidth, -height);
+    context.lineTo(halfWidth, -height);
+    context.lineTo(halfWidth, -triangleHeight);
+    context.lineTo(0, 0);
+    context.lineTo(-halfWidth, -triangleHeight);
+    context.closePath();
+  },
+};
+
 /** Defines an SVG gradient with id `id` based on the provided `controlPoints` */
 const ControlPointGradientDef: React.FC<{ controlPoints: ControlPoint[]; id: string }> = ({ controlPoints, id }) => {
   const range = controlPoints[controlPoints.length - 1].x - controlPoints[0].x;
   return (
     <defs>
-      <linearGradient id={id} gradientUnits="objectBoundingBox" spreadMethod="pad" x1="0%" y1="0%" x2="100%" y2="0%">
+      <linearGradient id={id} gradientUnits="objectBoundingBox" spreadMethod="pad" x2="100%">
         {controlPoints.map((cp, i) => {
-          const offset = "" + ((cp.x - controlPoints[0].x) / range) * 100 + "%";
+          const offset = ((cp.x - controlPoints[0].x) / range) * 100 + "%";
```

Comment:
how about a backtick formatted string here, to defeat the evil "number plus string" expression?